src/packages/client/src/runtime/utils/printStack.ts

```diff
@@ -77,13 +77,16 @@ function parseStack({
   const trace = stack.reverse().find((t) => {
     return (
       t.file &&
+      t.file !== '<anonymous>' &&
       !t.file.includes('@prisma') &&
       !t.file.includes('getPrismaClient') &&
+      !t.file.startsWith('internal/') &&
       !t.methodName.includes('new ') &&
       !t.methodName.includes('_getCallsite') &&
       t.methodName.split('.').length < 4
```

Comment:
These conditions probably all make sense somehow but they are a bit mysterious 🤔 

What is `t.file !== '<anonymous>'`? a Node-API thing only maybe?

And what is `!t.file.startsWith('internal/') &&` I don't remember files starting like this 🤔

Reply:
Would also love to know this, @williamluke4

Reply:
So what I can gather is that `internal/` are internal nodejs files that are returned in the stack trace and `<anonymous>` seems to happen when there is a `try catch`.